### DISCUSSION TITLE
how to change 'ZITADEL' name in the web console page title?

### DISCUSSION THREAD

**Discussion by pschiffe** [zitadel/zitadel#8155] [Q&A] [answered]
Hello, when I'm in the web console of ZITADEL, the webpage has title `<title>ZITADEL • Console</title>`. Is there a way to change the ZITADEL name to the org name for example, or custom text?

Thank you.

**Comment by fforootd** [comment]
ATM we do not have a out of the box way to change this.

But you could fork zitadel and change this line https://github.com/zitadel/zitadel/blob/main/console/src/index.html#L5

**Reply by pschiffe** [reply]
I see, thank you. Shall I open an issue for this?

**Reply by fforootd** [reply]
If think that is a good idea, yes.

**Reply by pschiffe** [reply, marked as answer]
Thank you, reported here: https://github.com/zitadel/zitadel/issues/8158
